package.json

```
@@ -176,14 +176,14 @@
   },
   "scripts": {
     "android": "react-native run-android",
-    "build:android": "sh scripts/build.sh apk",
-    "build:android-unsigned": "sh scripts/build.sh apk unsigned",
-    "build:ios": "sh scripts/build.sh ipa",
-    "build:ios-sim": "sh scripts/build.sh ipa simulator",
-    "build:ios-unsigned": "sh scripts/build.sh ipa unsigned",
+    "build:android": "bash scripts/build.sh apk",
```

Oh!! Interesting, what about this instead (as we used to have it)

```suggestion
    "build:android": "./scripts/build.sh apk",
```

---

That works but requires the files to be executable which they should be as far as I can tell. But was there a reason for changing them in https://github.com/mattermost/mattermost-mobile/pull/7263?

---

Reason was me trying to be clever, nothing more

---

Oh well, then we can simply revert that portion and it should work. I'll send an update :)